src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -75,6 +75,14 @@ cd "$__TOOLS_DIR/portableTargets"
 "$__DOTNET_CMD" restore --source https://www.myget.org/F/dotnet-buildtools/ --packages "$__TOOLS_DIR/portableTargets/packages/"
 cp -R "$__TOOLS_DIR/portableTargets/packages/Microsoft.Portable.Targets/${__PORTABLETARGETS_VERSION}/contentFiles/any/any/." "$__TOOLRUNTIME_DIR/."
 
+# Temporary workaround for CLI #710: copy native artifacts that weren't during dotnet publish.
+# This must be in sync with tool-runtime/project.json.
+__PACKAGE_DIR="$HOME/.nuget/packages"
+cp "$__PACKAGE_DIR/Microsoft.Build.Targets/0.1.0-preview-00017/runtimes/any/native/"* "$__TOOLRUNTIME_DIR"
+cp "$__PACKAGE_DIR/Microsoft.Net.Compilers.netcore/1.2.0-beta1-20160122-02/runtimes/any/native/"* "$__TOOLRUNTIME_DIR"
```

Comment:
My OSX testing has been that the package is installed with the actual package ID as the path, not what's requested, so it's `Microsoft.Net.Compilers.netcore` for me even though it shows up in the lockfile as `NetCore`. But I don't know if this is the same on each platform, etc.

This seems like another reason to avoid this workaround.